content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Changing the Kubernetes package repository
+min-kubernetes-server-version: 1.24
+content_type: task
+weight: 120
+---
+
+<!-- overview -->
+
+This page explains how to switch from one to another Kubernetes package repository
+upon upgrading from one to another Kubernetes minor release. Unlike Google-hosted
+repositories, the Kubernetes community-owned package repositories are structured
+in a way that there's a dedicated package repository for each Kubernetes minor version.
+
+
+For more information about the Kubernetes community-owned package repositories,
+see the ["pkgs.k8s.io: Introducing Kubernetes community-owned package repositories"](TBD)
+blog post.
+
+## {{% heading "prerequisites" %}}
```

Comment:
I think you can simplify the text by merging the _prerequisites_ and the first step.

Think about it this way: if the first step is to check if the repos are in use, then it is not really a prerequisite to start the procedure.

Reply:
We should keep this heading. Maybe the first step becomes a subheading?

Reply:
> Maybe the first step becomes a subheading?

I like this idea!